### PR TITLE
Add content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-version.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translate content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-version.md

### CODE REVIEW

**Review by Sea-n — COMMENTED** [review]
Please refer to [Netlify build log](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6263de226aa1c400097be32c#L4008), feel free to `/unhold` yourself when it's resolved.

/hold

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-version.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- jbeda
+title: kubeadm version
+content_type: conceito
+weight: 80
+---
+<!-- overview -->
+Este comando exibe a versão do kubeadm.
+
+<!-- body -->
+{{< include "generated/kubeadm_version.md" >}}
```
[the remark this answers is not in the document]
Comment (MrErlison) [reply content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-version.md:13]: Thanks @Sea-n . Now I understand what you say. /unhold

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-version.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: kubeadm version
+content_type: conceito
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-version.md:3]: Keep "concept" here.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Minor change in front matter. After updated will be a LGTM.

**Review by rikatz — APPROVED** [review]
/lgtm
/approve
/unhold
Thanks

**Pull request merged**